vterm.el

```diff
@@ -848,6 +849,19 @@ the called functions."
          (or (= pt prompt-pt)
              (string-blank-p (buffer-substring-no-properties pt prompt-pt))))))
 
+(defun vterm-bol ()
```

Comment:
I think it would be better to call this function `vterm-beginning-of-line`, following Emacs' own `beginning-of-line`.

Then, I'd modify the docstring with
```emacs-lisp
"Move point to the beginning of the line.

Move the point to the first character after the shell prompt on this line.
If the point is already there, move to the beginning of the line.
Effectively toggle between the two positions.
```

Then, I still don't understand exactly how the prompt is detected... I think it would be useful to explain this in the README and in some docstrings (if it requires setup from the shell side).

Reply:
I renamed `vterm-bol` to `vterm-beginning-of-line`, And docstring is updated .